PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -203,94 +203,113 @@ fi
 
 source ${HCPPIPEDIR}/global/scripts/log.shlib  # Logging related functions
 source ${HCPPIPEDIR}/global/scripts/opts.shlib # Command line option functions
+source ${HCPPIPEDIR}/global/scripts/mppmodecheck.shlib  # Check MMP mode requirements
 
 # ------------------------------------------------------------------------------
 #  Usage Description Function
 # ------------------------------------------------------------------------------
 
 show_usage() {
-	cat <<EOF
+  cat <<EOF
 
 PreFreeSurferPipeline.sh
 
 Usage: PreeFreeSurferPipeline.sh [options]
 
-  --path=<path>        Path to study data folder (required)
-					   Used with --subject input to create full path to root
-					   directory for all outputs generated as path/subject
-  --subject=<subject>  Subject ID (required)
-					   Used with --path input to create full path to root
-					   directory for all outputs generated as path/subject
-  --t1=<T1w images>    An @ symbol separated list of full paths to T1-weighted
-					   (T1w) structural images for the subject (required)
-  --t2=<T2w images>    An @ symbol separated list of full paths to T2-weighted
-					   (T2w) structural images for the subject (required)
```

Comment:
Should we mention/include "NONE" as an option here?

Reply:
I think it would be good for the description to say something like "(required for hcp-style data, can be NONE for legacy-style data)"

Reply:
I added Tim's line.